docs/_docs/upgrading/3-to-4.md

```diff
@@ -21,7 +21,32 @@ gem update jekyll
 
 ---
 
-  *Insert sections here*
+### Exclusion & Inclusion changes
+
+We've enhanced our default exclusion array. It now looks like the following:
+
+```
+.sass-cache/
+.jekyll-cache/
+gemfiles/
+Gemfile
+Gemfile.lock
+node_modules/
+vendor/bundle/
+vendor/cache/
+vendor/gems/
+vendor/ruby/
+```
+
+What's new is that this array __cannot be overriden anymore__. You can forcibly include
+directories or files that have been excluded by default, though. This would look like this:
+
+```yml
+include:
+  - node_modules/.bin/**
```

Comment:
umm.. actually, I haven't tested if `include` works as expected with wildcards..

Reply:
oh! should probably do that then :P

Reply:
@oe Local tests show that wildcards do not work with includes yet.. (Can you confirm..?)
But I feel fixing that is outside the scope of the current PR...

Reply:
yeah, i'm just going to change the example in this scenario